about/index.html

```diff
@@ -0,0 +1,146 @@
+            </h2>
+          </div>
+          <div class="col-sm-12">
+            <h3 class="text-center font-lato fs-6 p-3 border-accent border-1 bg-white mt-2 mb-3 border-solid">
+              The virtual conference agenda will now be spread out over the week of November 22nd with a shorter 4-5 hours per day program, to ensure that content is delivered in more convenient segments. The program will include three days of pre-conference workshops and three days of conference proceedings.
+            </h3>
+          </div>
+          <div class="col-sm-6">
+            <h4 class="display-6 fs-6 mb-3 text-center">
+              Please, contact us per Email for any further questions about the WONCA 2021 World Conference!
+            </h4>
+            <p class="text-center text-lowercase font-coco fs-6 lh-1 text-decoration-underline">
+              contact@wonka.com
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="about py-5">
+    <div class="wrapper container-sm">
+      <h2 class="heading text-dark text-center text-capitalize mb-5">WONCA 2021 <br> World Conference Logo</h2>
+      <p class="text-center mb-5">The logo of WONKA 2021 World Conference was decided through a logo competition from 8.June to 7.July</p>
+      <img class="w-small w-75 d-flex mx-auto" src="../images/logo.png" alt="">
+    </div>
+  </section>
+
+  <section class="show py-5 border-top">
+    <div class="wrapper container-sm">
+      <h2 class="heading text-dark text-center text-capitalize mb-5">See Past Wonka Conferences</h2>
+      <p class="text-center mb-5">Take a look at the last 2 WONCA conferences that took place in Berlin and in keyoto</p>
+      <div class="d-flex flex-lg-row flex-column gap-3">
+        <img class="show-image d-flex mx-auto" src="http://www.familydoctorseurope2020.org/resource/wonca2020/files/WONCA2020VC_web_header-3.jpg" alt="">
+        <img class="show-image d-flex mx-auto" src="https://www.globalfamilydoctor.com/site/DefaultSite/filesystem/images/News/2019%20News/06%20June/kyoto%20YDM%20Preconference.jpg" alt="">
+      </div>
+    </div>
+  </section>
+
+  <section class="partners bg-dark py-5">
+    <div class="wrapper container-sm">
+      <h2 class="heading text-light text-center text-capitalize mb-5">Partners</h2>
+        <div class="d-flex flex-md-row flex-sm-column flex-column gap-2 justify-content-center align-items-center">
+          <img class="w-small" src="../images/em.jpg" alt="">
+          <img class="w-small" src="../images/wonca.svg" alt="">
+        </div>
+    </div>
+  </section>
+
+  <footer class="footer px-2 py-4 d-flex gap-3 align-items-center">
+    <img class="w-20 flex-1" src="../images/cc.svg" alt="">
+    <div class="copyrights flex-grow-3 flex-shrink-3 w-50">
+      <p class="mb-0">
+        2021 WONCA World Conference. All rights reserved.
+      </p>
+    </div>
+  </footer>
```

Comment:
- [ ] The footer contains a logo aligned to the left and text aligned to the right but on your page, the logo and the text are aligned on the left. Please fix that on both pages.